newIDE/app/src/UI/Theme/CreateTheme.js

```diff
@@ -0,0 +1,249 @@
+        marginTop: -9,
+        marginBottom: -9,
+      },
+    },
+    // Use non rounded buttons
+    MuiButton: {
+      root: {
+        borderRadius: 0,
+        fontWeight: 400, // Lower a bit the weight of buttons
+      },
+    },
+  };
+}
+
+export function getThemeMode(color: string, contrastText: string) {
+  return {
+    light: lighten(color, 0.05),
+    main: color,
+    dark: darken(color, 0.05),
+    contrastText: contrastText,
+  };
+}
+
+export function createGdevelopTheme(
+  styles: any,
+  rootClassNameIdentifier: string,
+  paletteType: string
+) {
+  return {
+    gdevelopTheme: {
+      palette: {
+        canvasColor: styles['ThemeSurfaceCanvasBackgroundColor'],
+      },
+      message: {
+        warning: styles['ThemeMessageWarningColor'],
+        error: styles['ThemeMessageErrorColor'],
+        valid: styles['ThemeMessageValidColor'],
+      },
+      toolbar: {
+        backgroundColor: styles['ThemeSurfaceWindowBackgroundColor'],
+        separatorColor: styles['ThemeToolbarSeparatorColor'],
+      },
+      closableTabs: {
+        fontFamily: styles['GdevelopFontFamily'],
+        containerBackgroundColor: styles['ThemeSurfaceWindowBackgroundColor'],
+        backgroundColor: styles['ThemeClosableTabsDefaultBackgroundColor'],
+        textColor: styles['ThemeClosableTabsDefaultColor'],
+        selectedBackgroundColor:
+          styles['ThemeClosableTabsSelectedBackgroundColor'],
+        selectedTextColor: styles['ThemeClosableTabsSelectedColor'],
+        ...closeableTabSizeOverrides,
+      },
+      imageThumbnail: {
+        selectedBorderColor: styles['ThemeSelectionBackgroundColor'],
+      },
+      list: {
+        itemsBackgroundColor:
+          styles['ThemeSurfaceAlternateCanvasBackgroundColor'],
+      },
+      searchBar: {
+        backgroundColor: styles['ThemeSurfaceAlternateCanvasBackgroundColor'],
+      },
+      listItem: {
+        groupBackgroundColor: styles['ThemeSurfaceWindowBackgroundColor'],
+        groupTextColor: styles['ThemeListItemGroupTextColor'],
+        deprecatedGroupTextColor:
+          styles['ThemeListItemGroupTextDeprecatedColor'],
+        separatorColor: styles['ThemeListItemSeparatorColor'],
+        selectedBackgroundColor: styles['ThemeSelectionBackgroundColor'],
+        selectedTextColor: styles['ThemeSelectionColor'],
+        rightIconColor: styles['ThemeRightIconColor'],
+        selectedRightIconColor: styles['ThemeRightIconColor'],
+        errorTextColor: styles['ThemeListItemErrorColor'],
+        warningTextColor: styles['ThemeListItemWarningColor'],
+        selectedErrorBackgroundColor: styles['ThemeListItemErrorColor'],
+        selectedWarningBackgroundColor: styles['ThemeListItemWarningColor'],
+      },
+      emptyMessage: {
+        shadowColor: styles['ThemeMessageEmptyShadowColor'],
+      },
+      ...getRootClassNames(rootClassNameIdentifier),
+      logo: {
+        src: 'res/GD-logo-big.png',
+      },
+      gdevelopIconsCSSFilter: '',
```

Comment:
This should be changed according to the theme? See the existing Solarized Dark and Nord theme, they have filters set ;)

Reply:
Oops! Got it, now it's set ✅